spec/index.html

```diff
@@ -1863,9 +1865,10 @@ <h3>Overview</h3>
         this by creating deterministic paths that emanate out from the blank
         node through any other adjacent blank nodes.</p>
 
-      <p>Ultimately, the algorithm selects a shortest <a>gossip path</a>,
-        distributing canonical identifiers to the unlabeled blank nodes
-        in the order in which they appear in this path.
+      <p>Ultimately, the algorithm selects a shortest <a>gossip path</a>
```

Comment:
Unless there might be multiple gossip paths of the same length... in which
case, something about that should also be said.
```suggestion
      <p>Ultimately, the algorithm selects the shortest <a>gossip path</a>
```